sys/include/event/periodic_callback.h

```diff
@@ -40,6 +40,17 @@ typedef struct {
     event_callback_t event;     /**< callback event portion */
 } event_periodic_callback_t;
 
+/**
+ * @brief   Get user context from Periodic Callback Event
+ *
+ * @param[in]   event           event_periodic_callback object to initialize
+ * @return      User supplied argument to the event
+ */
+static inline void *event_periodic_callback_get_arg(event_periodic_callback_t *event)
+{
+    return event->event.arg;
+}
```

Comment:
To me,

```diff
-    arg = ev->event.arg;
+    arg = event_periodic_callback_get_arg(ev);
```

does not feel too much of a win. But it may provide some flexibility for future refactoring.